assets/xml/overlays/ovl_Arrow_Fire.xml

```diff
@@ -0,0 +1,11 @@
+<Root>
+    <File Name="ovl_Arrow_Fire" BaseAddress="0x80865d10" RangeStart="0x9D0" RangeEnd="0x1DF0" Segment="128">    
+        <Texture Name="sFireArrow1Tex" OutName="fire_tex_1" Format="ia8" Width="32" Height="64" Offset="0x9D0"/>
+        <Texture Name="sFireArrow2Tex" OutName="fire_tex_2" Format="ia8" Width="32" Height="64" Offset="0x11D0"/>
+        <Array Name="sFireArrowVtx" Count="43" Offset="0x19D0">
+            <Vtx/>
+        </Array>
+        <DList Name="sFireArrowTextureDL" Offset="0x1C80"/>
```

Comment:
in the same vein, this isn't just setting the texture but also setting up the render mode (including blender), combiner and geometry mode,
so I would call this kind of dlist "SetupDL", with the idea it sets up the "drawing parameters" of the geometry that eventually comes next

I would completely agree a texture is more than an image and such a dlist could be viewed as a "texture", but we decided to call images "textures" so for consistency we can't have two definitions of "texture" 😅

Reply:
would it be fair to call that a MaterialDL and the other a ModelDL? I agree with geometry as well but model is a bit shorter and gets the same point across i think

Reply:
Material is correct.